ngraph/frontend/paddlepaddle/include/paddlepaddle_frontend/place.hpp

```diff
@@ -0,0 +1,206 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <frontend_manager/frontend_manager.hpp>
+#include <paddlepaddle_frontend/exceptions.hpp>
+
+namespace paddle
+{
+    namespace framework
+    {
+        namespace proto
+        {
+            class OpDesc;
```

Comment:
Do you have cycle dependency for all classes for which you use forward declaration?

Reply:
No, just full definition of this class is not needed for this header file to be compilable